### PR TITLE
add config and variable

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1937,3 +1937,10 @@ set tidb_slow_log_threshold = 200;
 - 是否持久化到集群：是
 - 默认值：`ON`
 - 这个变量用于控制计算窗口函数时是否采用高精度模式。
+
+### `tidb_stats_cache_mem_quota`
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 默认值：`0`
+- 这个变量用于控制 TiDB 内部用来维护统计信息的内存使用量上限。
```
Comment (TomShawn) [inline system-variables.md:1946]: 能否介绍得详细一些呢？对于取值是否有建议？或是链接到相关文档？

Comment (chrysan) [inline system-variables.md:1946]:
```suggestion
- 这个变量用于控制 TiDB 内部统计信息缓存的内存使用量上限。
```

Comment (chrysan) [reply system-variables.md:1946]: 维护有点歧义，analyze 也是维护统计信息。

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -465,6 +465,11 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：false
 + 该配置项可以控制系统变量 [`tidb_enforce_mpp`](/system-variables.md#tidb_enforce_mpp-从-v51-版本开始引入) 的初始值。例如，当设置该配置项为 true 时，`tidb_enforce_mpp` 的默认值为 ON。
 
+### `enable-stats-cache-mem-quota`
+
++ 用于控制 TiDB 是否开启维护统计信息的内存上限。
```
Comment (chrysan) [inline tidb-configuration-file.md:470]:
```suggestion
+ 用于控制 TiDB 是否开启统计信息缓存的内存上限。
```

**Review by chrysan — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -1937,3 +1937,10 @@ set tidb_slow_log_threshold = 200;
 - 是否持久化到集群：是
 - 默认值：`ON`
 - 这个变量用于控制计算窗口函数时是否采用高精度模式。
+
+### `tidb_stats_cache_mem_quota`
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 默认值：`0`
+- 这个变量用于控制 TiDB 内部统计信息缓存的内存使用量上限。
```
Comment (TomShawn) [inline system-variables.md:1946]:
```suggestion
- 这个变量用于控制 TiDB 内部统计信息缓存使用内存的上限。
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: system-variables.md [hunk 4]
```diff
@@ -1937,3 +1937,10 @@ set tidb_slow_log_threshold = 200;
 - 是否持久化到集群：是
 - 默认值：`ON`
 - 这个变量用于控制计算窗口函数时是否采用高精度模式。
+
+### `tidb_stats_cache_mem_quota`
```
Comment (TomShawn) [inline system-variables.md:1941]:
```suggestion
### `tidb_stats_cache_mem_quota` <span class="version-mark">从 v6.1.0 版本开始引入</span>
```

File: tidb-configuration-file.md [hunk 5]
```diff
@@ -465,6 +465,11 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：false
 + 该配置项可以控制系统变量 [`tidb_enforce_mpp`](/system-variables.md#tidb_enforce_mpp-从-v51-版本开始引入) 的初始值。例如，当设置该配置项为 true 时，`tidb_enforce_mpp` 的默认值为 ON。
 
+### `enable-stats-cache-mem-quota`
```
Comment (TomShawn) [inline tidb-configuration-file.md:468]:
```suggestion
### `enable-stats-cache-mem-quota` <span class="version-mark">从 v6.1.0 版本开始引入</span>
```

**Pull request merged**